File: codedex/legends_python/python/control_flow/ph_levels.py
"""
Instructions
In chemistry, pH is a scale used to specify the acidity or basicity of a liquid. 🧪

Create a ph_levels.py program that checks whether a pH level is basic, acidic, or neutral.

First, create a ph variable and ask the user for a value between 0 and 14.

Write an if, elif, else statement that:

If ph is greater than 7, output "Basic".
If ph is less than 7, output "Acidic".
Else, output "Neutral".

"""
# ph = 1
# ph = 12
# ph = 7
ph = int(input('Enter a pH level (0-14): '))

if ph > 7:
  print('Basic')
elif ph < 7:
  print('Acidic')
else:
  print('Neutral')

File: codedex/origins_trilogy/js/vanilla-js/conditionals/ph-levels.js
"""

Instructions
In chemistry, pH is a scale used to specify the acidity or basicity of a fluid.

Create a ph-levels.js program that checks whether a pH level is basic, acidic, or neutral.

First, create a variable called ph and give it a value between 0 and 14.

Write an if/else if/else statement that:

If ph is greater than 7, output “Basic”.
Else if ph is less than 7, output “Acidic”.
Else, output “Neutral”.

"""



let ph = 5;

if (ph > 7) {
  console.log('Basic');
} else if (ph < 7) {
  console.log('Acidic');
} else {
  console.log('Neutral');
}
